src/Service/PageListGetter.php

```diff
@@ -76,6 +76,26 @@ public function getFromWhatLinksHere( $pageName, $extraParams = [] ) {
 		return $this->runQuery( $params, 'glhcontinue', 'pages' );
 	}
 
+	/**
+	 * Get all pages that are linked to from the given page..
+	 *
+	 * @link https://www.mediawiki.org/wiki/API:Links
+	 * @uses PageListGetter::runQuery()
+	 *
+	 * @param string $pageName The page name
+	 * @param string[] Any extra parameters to use: pltitles, plnamespace, pldir, pllimit
```

Comment:
Don't these extra params need a `g` prefix if they're being used with the generator? (I might be wrong, I've not actually tested this.)